views/projects/show.erb

```diff
@@ -3,43 +3,86 @@
     <a href="/projects/<%= @project.id %>/edit">edit</a>
   </h2>
   <p>
-    Client Id: <%= @project.client_id %> <br>
-    Resources: <%= @project.resources %> <br>
+    <% for @client in @clients %>
```

Comment:
We really should only ever have one client per project, so we don't need a full loop here, just take the first in the clients array.